client/reader/stream/index.jsx

```diff
@@ -518,29 +493,16 @@ class ReaderStream extends Component {
 				/>
 			);
 
-			let sidebarContent = null;
-			let tabTitle = translate( 'Sites' );
+			let sidebarContent = this.props.streamSidebar;
 
-			if ( isTagPage || isDiscoverTags ) {
-				sidebarContent = <ReaderTagSidebar tag={ tag } />;
-				tabTitle = translate( 'Related' );
-				baseClassnames = classnames( 'tag-stream__main', this.props.className );
-			} else if ( isSearchPage ) {
+			// TODO: use `streamSidebar` prop in search stream component, rather that using conditionals here.
+			if ( isSearchPage ) {
 				sidebarContent = (
 					<ReaderPopularSitesSidebar items={ items } followSource={ READER_SEARCH_POPULAR_SITES } />
 				);
-			} else if ( isDiscoverStream ) {
-				sidebarContent = (
-					<ReaderPopularSitesSidebar
-						items={ sites }
-						followSource={ READER_DISCOVER_POPULAR_SITES }
-					/>
-				);
-			} else {
-				sidebarContent = <ReaderListFollowedSites path={ path } />;
 			}
 
-			if ( excludesSidebar.includes( streamType ) ) {
+			if ( ! sidebarContent || streamType === 'search' ) {
```

Comment:
I don't think this will work because when on the search page and there is no search query the `streamType` is `custom_recs_sites_with_images` - this is what loads the popular posts/sites.

Reply:
See additional discussion in https://github.com/Automattic/wp-calypso/pull/78610/files#r1244546001